test/feature/StrongMode/Ellision.js

```diff
@@ -0,0 +1,15 @@
+// Options: --strong-mode
+
+'use strong'
+
+let a = [0, 1, 2, ];
+assert.equal(3, a.length);
+
+let [, b, , c] = [0, 1, 2, 3];
+assert.equal(1, b);
+assert.equal(3, c);
+
+// Not working ATM.
+// let d;
+// [, d] = [0, 1];
```

Comment:
It is not clear what to do about this? Conceptually it should be supported because it doesn't create holes. But since the pattern uses the same grammar as ArrayLiteral we will need to create a cover grammar for this... or do a second pass.